globalization/localization.md

```diff
@@ -10,301 +10,181 @@ position: 1
+>
+* Missing or non-translated messages in the UI
+* [Exceptions related to missing localization keys]({%slug common-kb-null-value-parameter-format%})
+>
+> Telerik supports the default English strings. The `resx` files for the other languages are provided as-is and you can use them as base for implementing your own.
 
-### Step 1: Enable the .NET Core Localization Services
+Make sure the `resx` files are defined as `EmbeddedResource` in the **Build Action** properties or in the `.csproj` file. Normally, the `Designer` file is generated automatically on every `resx` file save.
 
-Here is an example for enabling localization in the app:
-
->caption `Program.cs`
+>caption Sample resource file configuration in the .csproj file
 
 <div class="skip-repl"></div>
 
-````CS
-var builder = WebApplication.CreateBuilder(args);
-
-// ...
-
-// AddTelerikBlazor() registers the built-in service that includes only the default English labels.
-// The actual localizer for the Telerik components must be registered after the line below.
-builder.Services.AddTelerikBlazor();
-
-#region Localization Part 1
-
-// Register your Telerik component localizer after the built-in Telerik services above.
-builder.Services.AddSingleton(typeof(ITelerikStringLocalizer), typeof(SampleResxLocalizer));
-
-// Standard .NET localization code
-builder.Services.AddControllers();
-builder.Services.AddLocalization(options => options.ResourcesPath = "Resources");
-builder.Services.Configure<RequestLocalizationOptions>(options =>
-{
-    // The list of cultures that the app will support.
-    var supportedCultures = new List<CultureInfo>()
-            {
-                new CultureInfo("en-US"),
-                new CultureInfo("de-DE"),
-                new CultureInfo("es-ES"),
-                new CultureInfo("bg-BG")
-            };
-
-    // Set the default culture.
-    options.DefaultRequestCulture = new RequestCulture("en-US");
-
-    options.SupportedCultures = supportedCultures;
-    options.SupportedUICultures = supportedCultures;
-});
-
-#endregion Localization Part 1
-
-var app = builder.Build();
-
-#region Localization Part 2
-
-// Standard .NET localization code
-app.UseRequestLocalization(app.Services.GetService<IOptions<RequestLocalizationOptions>>().Value);
-app.MapControllers();
-
-#endregion Localization Part 2
+````XML
+  <ItemGroup>
+    <EmbeddedResource Update="Resources\TelerikMessages.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>TelerikMessages.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Resources\TelerikMessages.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>TelerikMessages.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+````
 
-// ...
+### Step 3: Implement ITelerikStringLocalizer Service
```

Comment:
Consider adding a definite or indefinite article here (aka "an ITelerik..." or "the ITelerik...")

Reply:
I also wondered about this, but as far as I know, headings can omit them for brevity and readability, so opt for this approach.